packages/angular/projects/ui-angular/src/lib/primitives/tab-item/tab-item.component.ts

```diff
@@ -10,4 +10,5 @@ export class TabItemComponent {
   @Input() id: string;
   @Input() labelledById: string;
   @Input() tabIndex: number;
+  @HostBinding('style.display') display = 'block'; // emulate div behavior
```

Comment:
Angular nuance here, `tab-item` wrapper needs this set so that it takes the proper height set by CSS.